src/makeInfinityAdder.js

```diff
@@ -5,6 +5,25 @@
  */
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  function adder(...nums) {
```

Comment:
The function `adder` is defined to accept a rest parameter `nums`, but according to the task description, the function should only accept a single argument at a time. You need to adjust the function to accept only one parameter.